File: Br.Com.Fiap.Gere.Residuo/Model/BairroModel.cs
namespace Br.Com.Fiap.Gere.Residuo.Model
{
    public class BairroModel
    {
        public int BairroId { get; set; }

        // Relacionamento com MoradorModel
        public List<MoradorModel> MoradoresDoBairro { get; set; }

        // Relacionamento com AgendaModel
        public List<AgendaModel> AgendasDeColetaDeLixoDoBairro { get; set; }

        public string BairroNome { get; set; }

        public int QuantidadeLixeiras { get; set; }

        public long PesoMedioLixeirasKg { get; set; }

        public int PercentualColetaLixoBairro { get; set; } = 100;

        public bool BairroEstaDisponivel { get; set; } = true;

        
    }
}

File: BDD_Tests/BDD-Gere-Residuo/src/test/java/model/bairro/BairroModel.java
package model.bairro;

import com.google.gson.annotations.Expose;
import lombok.Data;

@Data
public class BairroModel {

    @Expose(serialize = false)
    private int bairroId;
    @Expose(serialize = false)
    private int percentualColetaLixoBairro;
    @Expose(serialize = false)
    private boolean bairroEstaDisponivel;
    @Expose
    private String bairroNome;
    @Expose
    private int quantidadeLixeiras;
    @Expose
    private long pesoMedioLixeirasKg;
}
